lib/cogctl/actions/relays/delete.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Cogctl.Actions.Relays.Delete do
+  use Cogctl.Action, "relays delete"
+
+  def option_spec do
+    []
+  end
+
+  def run(options, args, _config, %{token: nil}=endpoint) do
+    with_authentication(endpoint, &run(options, args, nil, &1))
+  end
+  def run(_options, [], _config, _endpoint), do: display_arguments_error
+  def run(_options, args, _config, endpoint) do
+    do_delete(endpoint, args)
+  end
+
+  defp do_delete(endpoint, relay_names) do
+    Enum.each(relay_names, fn relay_name ->
```

Comment:
Agreed; however the way the code is written now, that `cogctl` invocation would exit with code `0`, but we'd want to return some non-zero exit code if any deletion failed.

Perhaps switching back to a `reduce` approach, letting it go all the way through the list of names, but letting the accumulator be the final status would be best. Either that or a `map` approach that looks at the returned values and returns `:error` if there are any `:error`s in the resulting list.